src/components/menu-bar/author-info.jsx

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import {FormattedMessage, injectIntl} from 'react-intl';
+import UserAvatar from './user-avatar.jsx';
+
+import styles from './author-info.css';
+
+const AuthorInfo = ({
+    className,
+    projectTitle,
+    userId,
+    username
+}) => (
+    <div
+        className={classNames(
+            className
+        )}
+    >
+        <div
```

Comment:
The outer wrapper and this one look like they could be combined.

Reply:
You're right. I had split them apart when trying to wrest control of the flex width from CSS, but they can be back together now.

Reply:
Done.